platform/ios/MGLMapView.mm

```diff
@@ -142,6 +142,7 @@ - (instancetype)initWithCoder:(NSCoder *)decoder
     if (self && [self commonInit])
     {
         self.styleURL = nil;
+        self.accessToken = [MGLAccountManager accessToken];
```

Comment:
No difference, but I guess we should get `-[MGLMapView setAccessToken:]` to bail on `nil`.